src/App.js

```diff
@@ -1,5 +1,39 @@
+import { Console } from '@woowacourse/mission-utils';
+import Racing from './Racing.js';
+import RacingManager from './RacingManager.js';
+import Validation from './Validation.js';
+
 class App {
-  async run() {}
+  async run() {
+    const validationClass = Validation;
+    const racingManager = new RacingManager(validationClass);
```

Comment:
```suggestion
#racingManager;

  constructor() {
    this.#racingManager = new RacingManager(Validation);  
  }
  async run() {
// racingManager -> this.#racingManager로 사용
```
이런 경우에는 굳이 validationClass에 담은 후에 Racing Manager에 넣는 것 보다 바로 생성자 파라미터에 담는게 좋아 보입니다!!
또한 Input을 받기 전에 사전 준비 작업 느낌이기 때문에 따로 run 외부의 생성자로 빼는 것은 어떠신 가요!?
이렇게 하시면 run의 책임 분산, 캡슐화 등등 많은 이점이 있을 것 같아요!!

Reply:
오 어차피 인스턴스를 생성할 필요가 없으니, 바로 담는 것이 좋아보이네요. 제안 감사합니다!
저도 모르게 `run` 메서드를 다른 파일에서 만든 로직 조각을 끼워맞추는 퍼즐판? 정도로 생각해서 메서드를 마구잡이로 넣고 있었는데, 생각해보니 `run`도 엄연한 함수네요.. 분리를 고려해봐야겠습니다. 감사합니다!